pkg/modeling/modeling_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"reflect"
 	"testing"
 
-	rbt "github.com/emirpasic/gods/trees/redblacktree"
+	rbtree "github.com/emirpasic/gods/trees/redblacktree"
```

Comment:
```suggestion
	"github.com/emirpasic/gods/trees/redblacktree"
```
It's ok to omit the alias.

Reply:
Got it.